### PR TITLE
Add release log and bump version

### PR DESCRIPTION
**Pull request by zhangjustin**
## Summary of changes

Release log and bump version.

/fyi:  @mark-juwai , @tjoelsson : Please review.

### CODE REVIEW

**Review by mark-juwai — APPROVED** [review]

File: CHANGES [hunk 1]
```diff
@@ -3,6 +3,13 @@ Zask Changelog
 
 Here you can see the full list of changes between each Zask release.
 
+Version 1.9.8
+-----------
+
+released on July 3th 2019
```
Comment (tjoelsson) [inline CHANGES]:
```suggestion
released on July 3rd 2019
```

File: CHANGES [hunk 2]
```diff
@@ -3,6 +3,13 @@ Zask Changelog
 
 Here you can see the full list of changes between each Zask release.
 
+Version 1.9.8
+-----------
```
Comment (tjoelsson) [inline CHANGES]:
```suggestion
-------------
```

File: CHANGES [hunk 3]
```diff
@@ -3,6 +3,13 @@ Zask Changelog
...
+
+released on July 3th 2019
+
+* Add origin access to thread local and bring it to next service
```
Comment (tjoelsson) [inline CHANGES]: access key

**Comment by zhangjustin** [comment]
Updated, please review.

**Review by tjoelsson — APPROVED** [review]

**Review by mark-juwai — APPROVED** [review]

**Pull request merged**